Standards/CCSMB-3.md

```diff
@@ -0,0 +1,107 @@
+# CCSMB-3 bimg File Format
+
+*Author: SkyTheCodeMaster*
```

Comment:
I feel like I should be included here, as I wrote a great part of it in [this PR](https://github.com/SkyTheCodeMaster/bimg/pull/4), originating from my fork, and merged to Sky's original repo.

Reply:
I think all the contributors in SkyTheCodeMaster/bimg should be listed here.

Reply:
I do agree